src/components/Layout/Footer/Footer.module.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../variables.scss';
+
+.c-footer {
+    background-color: $color-blue;
+    position: absolute;
+    bottom: 0;
+    color: $color-white;
+    width: 100%;
+    height: $space-xl;
+    text-align: center;
+    padding: $space-s 0;
```

Comment:
This part is good in general, but I suggest that you try to group css properties in some logical way, for example: 
```
width: 100%;
height: $space-xl;

position: absolute;
bottom: 0;

padding: $space-s 0;
text-align: center;

background-color: $color-blue;
color: $color-white;
```